packages/flutter_test/lib/src/matchers.dart

```diff
@@ -222,9 +222,11 @@ const Matcher isNotInCard = _IsNotInCard();
 
 /// Asserts that the object represents the same color as [color] when used to paint.
 ///
-/// Specifically this matcher checks the object is of type [Color] and its [Color.value]
-/// equals to that of the given [color].
-Matcher isSameColorAs(Color color) => _ColorMatcher(targetColor: color);
+/// Specifically this matcher checks the object is of type [Color] and its color
+/// components fall below the specified delta specified by [threshold].
```

Comment:
maybe drop one of the "specified"s?

```suggestion
/// components fall below the delta specified by [threshold].
```

Reply:
done